src/App.tsx

```diff
@@ -1,10 +1,48 @@
+/* eslint-disable max-len */
+/* eslint-disable react/jsx-no-comment-textnodes */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { FilterStatus, Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { ErrorNotification } from './components/ErrorNotification';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [filterStatus, setFilterStatus] = useState<FilterStatus>(
+    FilterStatus.ALL,
+  );
+  const [todoQuery, setTodoQuery] = useState<string>('');
+  const [todoError, setTodoError] = useState<string>('');
```

Comment:
Would be good to create `enum` for all possible errors and '' as a default value